package.json

```diff
@@ -64,6 +64,16 @@
           "default": false,
           "description": "Suppress warnings when sorting fails."
         },
+        "sort-imports.cache-package-json-config-checks": {
+          "type": "boolean",
+          "default": true,
+          "description": "If true, will cache calls to `import-sort-config`, thereby improving performance."
+        },
+        "sort-imports.default-sort-style": {
+          "type": "string",
+          "default": "module-scoped",
```

Comment:
Default should be `eslint` otherwise we're changing the default style for all current users.